cli/crates/server/src/servers.rs

```diff
@@ -270,12 +272,24 @@ async fn spawn_servers(
     Ok(())
 }
 
-fn export_embedded_files() -> Result<(), ServerError> {
-    let environment = Environment::get();
-
+/// Writes the assets to `Environment.user_dot_grafbase_path` based on following conditions:
+///
+/// * skip if environment variable `GRAFBASE_SKIP_ASSET_VERSION_CHECK` is set
+/// * write if environment variable `GRAFBASE_FORCE_EXPORT_FILES` is set
+/// * skip if version file in `Environment.user_dot_grafbase_path` and current version are same
+/// * write otherwise
+///
+/// # Errors
+///
+/// returns [`ServerError::ReadVersion`] if the version file for the extracted worker files cannot be read
+///
+/// returns [`ServerError::CreateCacheDir`] if the `Environment::user_dot_grafbase_path` cannot be created
+///
+/// returns [`ServerError::WriteFile`] if a file cannot be written into `Environment.user_dot_grafbase_path`
+pub fn export_embedded_files(user_dot_grafbase_path: &Path) -> Result<(), ServerError> {
```

Comment:
Is there a reason not to rely on the environment anymore? I mean, don't get me wrong, I have nothing against not using globals. :D Just wondering. :)